src/webui/static/app/status.js

```diff
@@ -42,16 +42,16 @@ tvheadend.status_subs = function(panel, index)
             totalProperty: 'totalCount',
             fields: [
                 { name: 'id', sortType: stype },
-                { name: 'hostname', sortType: stype },
+                { name: 'hostname', sortType: stypei },
```

Comment:
I'm not completely sure if hostname should be an integer or not, since it could be a domain name in some cases (or not? NOT tested)